css/datagrid/datagrid.css

```diff
@@ -83,42 +83,160 @@
     --highcharts-highlight-color-10: rgb(58, 63, 71);
 }
 
-/* stylelint-disable max-line-length */
+.hcdg-theme-default {
+    --border-radius: 5px;
+    --color: var(--highcharts-neutral-color-100);
+    --padding: 8px;
+    --border-width: 1px;
+    --border-color: var(--highcharts-neutral-color-20);
+    --header-font-weight: bold;
+    --background-color: var(--highcharts-background-color);
+    --row-even-background-color: var(--highcharts-neutral-color-3);
+    --header-row-background-color: var(--highcharts-neutral-color-5);
 
-/* ==== DataGrid Core ==== */
-.highcharts-datagrid-container * {
-    box-sizing: border-box;
-    font-family:
-        -apple-system,
-        BlinkMacSystemFont,
-        "Segoe UI",
-        Roboto,
-        Helvetica,
-        Arial,
-        "Apple Color Emoji",
-        "Segoe UI Emoji",
-        "Segoe UI Symbol",
-        sans-serif;
+    --row-hovered-background-color: var(--highcharts-highlight-color-10);
+    --column-hovered-background-color: rgba(0, 0, 255, 0.03);
+    --header-column-hovered-background-color: rgba(0, 0, 255, 0.07);
+
+    --row-synced-background-color: rgba(255, 0, 0, 0.1);
+    --column-synced-background-color: rgba(255, 0, 0, 0.06);
+    --header-column-synced-background-color: rgba(255, 0, 0, 0.1);
+
+    --header-text-align: center;
+
+    --credits-font-size: 0.6em;
+    --credits-text-align: right;
 }
 
+/* stylelint-disable max-line-length */
+
+/* ==== DataGrid Core ==== */
 .highcharts-datagrid-container {
+    --hcdg-table-border-radius: var(--border-radius, 0);
+
+    --hcdg-font-weight: var(--font-weight, normal);
+    --hcdg-font-size: var(--font-size, 1rem);
+    --hcdg-font-family: var(--font-family, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", sans-serif);
+    --hcdg-color: var(--color, #000);
```

Comment:
I would like to introduce an internal `--hcdg-default-color` variable that is `#000` or `#FFF`, based on system settings. Can be put inside the `highcharts-ligh`t and `highcharts-dark` on the top, if we don't want another prefers-color-scheme block. `--hcdg-default-color` should be used as fallback for the following:

```
--hcdg-color: var(--color, var(--hcdg-default-color));
--hcdg-caption-color: var(--caption-color, var(--hcdg-default-color));
--hcdg-description-color: var(--description-color, var(--hcdg-default-color));
--hcdg-table-border-color: var(--border-color, var(--hcdg-default-color));
--hcdg-credits-color: var(--credits-color, var(--hcdg-default-color));
```

Why? If users starts from scratch with no theme they should be able to see the actual content, also in darkmode. We only care about text and borders, and we should assume that they have a dark/light background-color on body or some parent element wrapping the datagrid.

Now, we could use `--highcharts-neutral-color-100`for simplicity, but it's just a "fluke accident" that this one is a neutral black/white. In other Highcharts color schemes it could be different, and I want it to be 100% neutral when they start from scratch.